eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -230,11 +230,61 @@ function Set-ApiViewCommentForPR {
   }
 }
 
-function Save-PackageProperties ($repoRoot, $serviceNames, $outputDirectory) {
-  $scriptPath = Join-Path $repoRoot "eng" "common" "scripts" "Save-Package-Properties.ps1"
-  $serviceDirectories = $serviceNames -split ","
+# Helper function used to create API review requests for Spec generation SDKs pipelines
+function Create-API-Review {
+  param (
+    [string]$apiviewEndpoint = "https://apiview.dev/PullRequest/DetectAPIChanges",
+    [string]$specGenSDKArtifactPath,
+    [string]$apiviewArtifactName,
+    [string]$buildId,
+    [string]$commitish,
+    [string]$repoName,
+    [string]$pullRequestNumber
+  )
+  . ${PSScriptRoot}\..\logging.ps1
+  $specGenSDKContent = Get-Content -Path $SpecGenSDKArtifactPath -Raw | ConvertFrom-Json
+  $language = ($specGenSDKContent.language -split "-")[-1]
```

Comment:
Is the language field the full pipeline or repo name? Is that way you are splitting on "-"?

Reply:
The language is the last part of the repo name